pkg/network/ovn_kubernetes.go

```diff
@@ -1258,16 +1258,19 @@ const (
 type targetZoneModeType struct {
 	// zoneMode indicates the target zone mode that CNO is supposed to converge to.
 	zoneMode InterConnectZoneMode
-	// "temporary", if true, indicates that the target zone mode is only temporary;
-	// it is used along with zoneMode=singlezone in order to temporarily switch to single-zone mode when upgrading
-	// from versions with no interconnect support (<=4.13). It has no effect if zoneMode=multizone.
-	temporary bool
 	// "configMapFound" indicates whether the interconnect configmap was found; when not found,
 	// the zone mode defaults to multizone.
 	configMapFound bool
 	// ongoingUpgrade is true when the configmap was pushed by CNO itself; it is used by status manager
```

Comment:
do we need to make updates to what ongoingUpgrade means? It seems like its also taking over the role that "temporary" was doing...

Reply:
Actually, since we've decided that we're not allowing a cluster to be permanently in single zone, I might as well remove "ongoingUpgrade" and just refer to `configMapFound`, since only CNO is supposed to push the configmap now... let me try it out!

Reply:
ack did you try it? all ok on this PR?

Reply:
yes, I retested everything after pushing the current changes: https://github.com/openshift/cluster-network-operator/pull/2076#issuecomment-1851982323